packages/ffe-form/less/input-group.less

```diff
@@ -22,40 +23,29 @@
         }
     }
 
-    > *:nth-last-child(1) {
-        margin-bottom: @ffe-spacing-lg;
-    }
+    > * {
```

Comment:
Er det riktig at denne ikke lenger skal targette siste child-element? Fordi sånn jeg oppfatter det nå, så blir det for liten space mellom hver input group?

Reply:
Dom her last child grejerna.. vist var det feilmedlningen?

Reply:
Den gamle løsningen her ga en margin til siste element uten feilmelding, som så ble flyttet til nest siste child med feilmelding slik at feilmeldingen havna inni marginen.

I stedet for å flyte rundt på denne marginen så har jeg satt en padding på selve input-group. som skaper plassen feilmeldingen havner inni

Reply:
Ok! :)